### DISCUSSION TITLE
PubPub font?

### DISCUSSION THREAD

**Discussion by katycroffbell** [knowledgefutures/pubpub#2148] [General]
What font is used for the main body text in pubs?

**Comment by gabestein** [comment]
Hello! It's the appropriately (?) named Skolar Latin (https://fonts.adobe.com/fonts/skolar)

**Comment by katycroffbell** [comment]
Cool, thanks! And correct me if I'm wrong, but the exported pdf uses a *different* font?! 

(FYI I'm trying to make tables as images so their formatting doesn't get messed up in the pdf, but if I use Skolar to match online, then it looks funky in the pdf because the body text is different)

**Comment by gabestein** [comment]
Yes, at the moment that's DejaVu Serif. The reason for that is that Skolar-Latin's encoding causes an [obscure error](https://bugs.chromium.org/p/chromium/issues/detail?id=108937) in Chromium, the browser engine used the generate the PDF, which causes PDFs to be unsearchable, and sometimes corrupt (see: https://github.com/pubpub/pubpub/issues/1640). We previously did not have much control over fonts on the machine used to generate PDFs, so we had to default to a system-installed TrueType font, the only available option for which was DejaVu. Now that we have more control, we'll likely try to unify the fonts in the coming months.

Re: tables, sorry I didn't get to that comment yet. To save you the hassle, I can take a look at PDF generation to see if there's a way we can better preserve table column and row spacing when it makes the transition to PDF. I think we probably can.

I'm out next week, so it will be the week after, and it likely won't be perfect since it is autogeneration after all and there are some limitations to what we can do. But I suspect we can improve things.

**Comment by katycroffbell** [comment]
Gotcha. I figured there was a reason for the discrepancy (there always is). I'm using dejavu sans so they match at least one font online and in the pdf, but having them be natively rendered would be much better than my images, for many reasons. 

thanks!
